conda-store-server/conda_store_server/storage.py

```diff
@@ -223,7 +224,7 @@ def get(self, key):
             return f.read()
 
     def get_url(self, key):
-        return os.path.join(self.storage_url, key)
+        return posixpath.join(self.storage_url, key)
```

Comment:
Do you remember what failed without this? The effect of this change will be converting Windows paths to posixpaths:

```py
>>> import posixpath
>>>
>>> posixpath.join("foo", "bar")
'foo/bar'
>>>
>>> import os
>>> os.path.join("foo","bar")
'foo\\bar'
```

Reply:
Windows backslash paths are not correct for file URLs. Without this change, some of the buttons in the UI don't work because they use `%5C` (URL encoded `\`) instead of `/`.